tests/phpunit/tests/theme/wpThemeHasThemeJson.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+require_once __DIR__ . '/base.php';
+
+/**
+ * Tests wp_theme_has_theme_json().
+ *
+ * @group theme_json
+ *
+ * @covers ::wp_theme_has_theme_json
+ */
+class Tests_Theme_WpThemeHasThemeJson extends WP_Theme_UnitTestCase {
+
+	/**
+	 * Test that it reports correctly themes that have a theme.json.
+	 *
+	 * @covers wp_theme_has_theme_json
```

Comment:
```suggestion
```
Since the class has the `@covers` annotation for `::wp_theme_has_theme_json`, the test methods don't need a `@covers` annotation.

🔢 Applies elsewhere in this class.